sysboundary/ionosphere.cpp

```diff
@@ -550,8 +550,10 @@ namespace SBC {
    ) {
       std::vector< std::array<int, 3> > closestCells = getAllClosestNonsysboundaryCells(technicalGrid, i,j,k);
       if (closestCells.size() == 1 && closestCells[0][0] == std::numeric_limits<int>::min() ) {
-         std::cerr << __FILE__ << ":" << __LINE__ << ":" << "No closest cells found!" << std::endl;
-         abort();
+	 //mismatch on fsgrid and mpigrid?
```

Comment:
This has nothing to do with this branch?

Reply:
Correct. Still needs fixing :)

Reply:
I'll work on that, I don't agree with the statement I heard that this is no longer a bug.